### ISSUE TITLE
user can make a guess that is not a word

### ISSUE THREAD

**Issue by shifoc** [siddhu33/discordle#3]

**Comment by siddhu33** [comment]
considering the words list is quite small and the bot supports a wide set of word sizes, I didn't want to lock word selections behind _another_ dictionary unless I had confidence that it was relatively comprehensive.

**Comment by shifoc** [comment]
fair enough. I also noticed that the bot marked duplicate letters in yellow even if the word contains this letter once.

**Comment by siddhu33** [comment]
yeah, I will have to update the match logic at some point to handle that case :) 
Thanks for bringing the issues up though, hope you are enjoying the bot otherwise!

**Comment by shifoc** [comment]
I am enjoying it very much, thank you!
